demos/OH-EXF-Kamakura/docs/images/Readme.md

```diff
@@ -0,0 +1,11 @@
+# Images
```

Comment:
I noticed there is no license specified in any of the files, its OK?

Reply:
No, it's not ok. Although we have a LICENSE file at the top of the repo, every file should have a Copyright statement with an SPDX license identifier. Thanks for pointing this out!

Reply:
I've opened Issue #15 to make sure this doesn't get lost.